integration/src/main/resources/testusers/levi.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "levi",
```

Comment:
I couldn't get Levi to work. I was getting "don't know GCP subject 123456789" from Sam and getting very odd failures of "levi.bonechewer@test.firecloud.org" not equal "Levi.Bonechewer@test.firecloud.org".

Since I determined that Bella was over quota, I was able to revert to Bella/Elijah. I would like to have Levi working as an alternate when debugging quota problems, but something is wrong. I logged into Terra as Levi and everything was fine, so the user is defined, etc. I have no clue; do you?